### PR TITLE
LRDOCS-7973 Salesforce Connector

### PR DESCRIPTION
**Pull request by JamesAGarcia**
([LRDOCS-7973](https://issues.liferay.com/browse/LRDOCS-7973))
This PR includes the incomplete Salesforce Connector article, without RSTs or imgs. Its completion awaits on final testing, determination of the connector's default behavior, and creation of its `.lpkg` file. Till then I have commented out its content.

### CODE REVIEW

File: docs/commerce/latest/en/add-ons-and-connectors/liferay-commerce-connector-to-salesforce.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Once purchased, you can [download]() and [deploy]() the Salesforce Connector to a Liferay instance as an `.lpkg` file. This file contains multiple Talend Jobs that map data fields in Liferay Commerce to data fields in Salesforce. Once deployed, Dispatch Tasks are automatically created for each Talend Job. They can then be configured and run manually via the Dispatch UI, or scheduled to run at specific time intervals.
+
+```note::
+   The Liferay Commerce Connector to Salesforce is compatible with Liferay DXP 7.1.x+.
+```
+
+## Supported Entities
+
+The following chart While some entity data can be transferred both ways (e.g., Account, Order), some Salesforce entity data can only be transferred to Liferay (e.g., Account Contact, Price Book).
```
Comment (jrhoun) [inline docs/commerce/latest/en/add-ons-and-connectors/liferay-commerce-connector-to-salesforce.md:18]: > The following chart While some entity data can be transferred both ways (e.g., Account, Order), some Salesforce entity data can only be transferred to Liferay (e.g., Account Contact, Price Book).

Seems like a typo in this sentence?

Comment (JamesAGarcia) [reply docs/commerce/latest/en/add-ons-and-connectors/liferay-commerce-connector-to-salesforce.md:18]: argh, yes. Somehow my brain passed over this. I'll fix it.

**Comment by JamesAGarcia** [comment]
@jrhoun, I updated the PR with the requested fix.

File: docs/commerce/latest/en/add-ons-and-connectors/liferay-commerce-connector-to-salesforce.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+| --- | --- | --- | --- |
+| Account | Account | ✔️ | ✔️ |
+| Account Address | Account | ✔️ | ✔️ |
+| Account Contact | User | ✔️ |  |
+| Price Book | Price List | ✔️ |  |
+| Price Book Entry | Price Entry | ✔️ |  |
+| Product | Product | ✔️ |  |
+| Order | Order | ✔️ | ✔️ |
+| OrderItem | Order Item | ✔️ | ✔️ |
+| OrderNote | Order Note | ✔️ | ✔️ |
```
Comment (jrhoun) [inline docs/commerce/latest/en/add-ons-and-connectors/liferay-commerce-connector-to-salesforce.md:30]: @JamesAGarcia can you replace these emojis w/ their unicode equivalents? Thanks.

**Comment by JamesAGarcia** [comment]
@jrhoun, I've replaced the emojis with `&#10004;`

**Comment by jrhoun** [comment]
Merged. Thanks. af0036d384e8efc589b5bdcaeb86f52dab9abbed

**Pull request closed without merging**